src/tokens/functional/color/data-vis.json5

```diff
@@ -26,11 +26,47 @@
               scopes: ['bgColor'],
             },
             'org.primer.overrides': {
-              dark: '{base.display.color.blue.0}'
-            }
+              dark: '{base.display.color.blue.0}',
+            },
+          },
+        },
+        'gradient-stop-1': {
```

Comment:
Is there a reason this is the full string instead of nested values for the token name?

Reply:
Good point, I adjusted it to `gradientStop` and nested `1` and `2` inside.